template/builders/azure.jinja2

```diff
@@ -0,0 +1,10 @@
+- name: {{builder.vars.name}}
+  type: azure-arm
```

Comment:
Is this a reasonable default for all of azure or should it be made a variable?

Reply:
it actually isn't "arm" with what we think it means, it is "Azure Resource Manager" in this case